README.md

```diff
@@ -46,6 +46,42 @@ bus.bind("order", {
 });
 ```
 
+## Environment Configuration Variables
+
+For more details and up to date information on Kafka specific configs check the [Kafka official documentation](https://kafka.apache.org/documentation/#configuration).
+
+| Name     | Type   | Mandatory | Default | Description |
+| -------- | ------ | ------- | ------- |----------- |
+| **BUS_ADAPTER** | `str`  | false | `kafkaRetry`    | RIPE Bus adapter |
+| **KAFKA_HOSTS** | `str`  | false | `localhost:9092`    | Kafka nodes available, separated by a `,` (comma).  |
```

Comment:
This seems like too much information, so let's just include the most important variables and that an operator would most likely want to change. Hosts is one of them and might be the only one